packages/cli/generation/ir-generator/src/converters/type-declarations/convertExampleType.ts

```diff
@@ -109,6 +109,9 @@ export function convertTypeExample({
                 wireValue: example,
             });
         },
+        undiscriminatedUnion: () => {
+            throw new Error("Unsupported!");
```

Comment:
can you say what's going on here in case this actually gets thrown
`Examples are not supported for undiscriminated unions.`

Ideally this is a fern check rule as well

Reply:
just put it as a place holder for now -- my thought was i'm going to need to fill this out once i support examples? wdyt

Reply:
Oh dope. Gonna be tricky to support I think because you’ll have to do the Jackson thing of figuring out if the example is valid. (IMO better to disallow then allow invalid example shapes)